classes/ClaytonTestClass2.cls

```diff
@@ -0,0 +1,322 @@
+/**
+	String body = '';
+    String x = '\n\tpublic void method(){'
+            +'\n\t\tInteger i = 0;'
+            +'\n\t\ti = 1;'
+            +'\n\t\ti=2;'
+            +'\n\t\ti=3;'
+        +'\n\t}';
+    for(Integer i = 0; i < 200;i++){
+        body+=x.replace('method','method'+i);
+    }
+    system.debug('>>>\n\n'+body);
+ */
+public class ClaytonTestClass2 {
+    public void methodA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodBL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCC(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCD(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCE(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCF(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCG(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCH(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCI(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodCL(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDA(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDB(){
+		Integer i = 0;
+		i = 1;
+		i=2;
+		i=3;
+	}
+	public void methodDC(){
+		Integer i = 0;
```

Comment:
This variable doesn't seem to have a very insightful name. Please give variables names that suggest their purpose.